### DISCUSSION TITLE
关于和RapidVideOCR配合的VSF

### DISCUSSION THREAD

**Discussion by joseph16388** [SWHL/RapidVideOCR#60] [Q&A] [answered]
和RapidVideOCR配合的VSF，字幕搜索软件的CPU占用经常是100%，请问有什么优化设置或者其他Sub Finder软件能替代VSF减小CPU占用吗？谢谢！

**Comment by SWHL** [comment, marked as answer]
这个软件我也不是太熟悉，它使开源的，可以去它的[开源社区](https://sourceforge.net/projects/videosubfinder/)找找，有没有相应参数。感觉可以调用GPU，CPU就会降下来。

这个项目开发者还是很活跃的。

**Reply by joseph16388** [reply]
> 这个软件我也不是太熟悉，它使开源的，可以去它的[开源社区](https://sourceforge.net/projects/videosubfinder/)找找，有没有相应参数。感觉可以调用GPU，CPU就会降下来。
> 
> 这个项目开发者还是很活跃的。

好的 感谢
